tests/Feature/NotesTest.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Feature;
+
+use App\Models\Note;
+use App\Models\User;
+use Illuminate\Support\Str;
+use Tests\TestCase;
+
+class NotesTest extends TestCase
+{
+    public function testNotesCanBeAccesed(): void
+    {
+        $user = User::factory()->create();
+        $this->actingAs($user);
+
+        $response = $this->get("/notes");
+
+        $response->assertStatus(200);
```

Comment:
```suggestion
   $user = User::factory()->create();
        $this->actingAs($user)
            ->get("/notes")
            ->assertStatus(200);
```

Please refactor rest of the test like above - it will improve readability.